packages/communication-react/review/communication-react.api.md

```diff
@@ -606,7 +606,7 @@ export type ChatCompositeIcons = Pick<CompositeIcons, 'MessageDelivered' | 'Mess
 // @public (undocumented)
 export interface ChatCompositeProps extends BaseCompositeProps<ChatCompositeIcons> {
     adapter: ChatAdapter;
-    onRenderMessage?: (messageProps: MessageProps, defaultOnRender?: DefaultMessageRendererType) => JSX.Element;
+    onRenderMessage?: (messageProps: MessageProps, defaultOnRender?: MessageRenderer) => JSX.Element;
```

Comment:
we defaultOnRenderer can we just call the parameter messageRenderer? 

our functions should still have a parameter that takes in an optional message renderer and they could pass in the default one if they want to but i dont think we should use the word default in the paramter?

Reply:
This function is a callback - so the parameter here is what we pass to developers - like message data + the default renderer for message data, return value is what developers wanna pass back to us

So defaultMessageRender is the right name cause it is passing our renderer for developers to consume